chapters/develop-organize/overview/reading/README.md

```diff
@@ -1,18 +1,50 @@
 ## Developing and Organizing Content
 
-Owner: Răzvan Nițu
-Reviewer: Răzvan Deaconescu
+Developing educational content is a time consuming activity that most educators would rather avoid.
+Typically, formal education institutions from similar fields have similar curriculums.
+As such, it would be much more time effective to share the educational materials used for one course rather than build them from scratch.
+To facilitate this, we propose a methodology for developing and organizing the educational materials such that the content is easily distributable, customizable, contributable and publishable.
+Ideally, once content has been created for a particular course or field of study, it will be usable by both institutions to teach the course or by self-taught individuals for self-study.
```

Comment:
```suggestion
Ideally, once content has been created for a particular course or field of study, it will be usable both by institutions to teach the course or by self-taught individuals for self-study.
```